README.md

```diff
@@ -1,75 +1,67 @@
-# KintoHub {name}-example
+# KintoHub Fastify Example
 
 ## Overview
-TODO: Cool description / goal of what technology you are bringing onto the plate with what language. This should be sweet and concise.
 
-[Live Example](live/example/on/kintohub)
+Fastify is one of the widely used web frameworks in Node.js. It can be built easily for microservice on KintoHub!
+
+[Live Example](https://8caf94bb-c2e5-45cf-898d-ca5b19954cda.api.master.kintohub.com/fastify-example/hello/world)
 
 __About KintoHub:__
 
 KintoHub aligns teams to ship & operate cloud native apps with ease. [Learn More](https://www.kintohub.com)
-## [Optional] Requirements
-
-- Ensure you have an account on [KintoHub](www.kintohub.com/sign-up)
-- [Optional] Required Kintohub features: [list features] (like pro sub features)
-- [Optional] specific packages needed for project to run
 
 ## Deployment
-1. Apply this template to your [Github](https://github.com/kintohub/example-template/generate)
-2. Create a [KintoBlockType](www.link.to.create.kintoblock.type.here) and include:
-
-You're now good to build! Click `Build Latest Commit`. Once complete, Click  Now click `Add To Project`.
-
-3. Create Project and include:
+1. Apply this template to your [Github](https://github.com/kintohub/fastify-example/generate)
+2. Create a [Microservice Block](https://docs.kintohub.com/docs/kintoblocks/microservices) on KintoHub
+3. Connect your GithubApp to KintoHub and select the Repository you just generated on your account.
+4. Set the **name** of your kinto block
+5. Choose `Node.js` as the **language** and `11.7.0` as the **version**
+6. Set the **Build Command** as `npm install`
+7. Set the **Start Command** as `npm start`
+8. Set the **Port** as `80`
 
+You're now good to go! Click **Create Microservice**. Now click **Build Latest Commit**
 
-You're ready to go live Click `Deploy`
+Once complete, Click *Add to Project*
```

Comment:
Would suggest **bold** for all buttons or elements to click.

Reply:
updated, gonna fix other example as well